chopper_generator/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.0.0-nullsafety.0
+
+- Null safety
```

Comment:
```suggestion
- Sound null safety support
```

Reply:
Or maybe let's go with 4.0? What do you think? :)